ggshield/cmd/hmsl/__init__.py

```diff
@@ -12,13 +12,25 @@
 from ggshield.cmd.hmsl.quota import quota_cmd
 
 
+class NaturalOrderGroup(click.Group):
+    """
+    Helper class to force the order of the commands in the help text.
+    Copied from https://github.com/pallets/click/issues/513#issuecomment-504158316
+    This will not work with python < 3.6.
```

Comment:
nit:
```suggestion
    """
    Helper class to force the order of the commands in the help text.
    Copied from https://github.com/pallets/click/issues/513#issuecomment-504158316
```
as we don't support Python 3.6

Reply:
I know we don't support <py3.8 but I wanted to emphasis the reason why it works and warn anyone willing to re-use this code on other projects